src/detection/card/items/class-item.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Typography } from '@material-ui/core'
+
+import { useStyles } from 'detection/card/items/style'
+import { useItem } from 'detection/card/use-item'
+
+export const ClassItem = () => {
```

Comment:
The `item` constant is unnecessary:
![image](https://user-images.githubusercontent.com/23569938/129885362-1a6edaca-556c-4850-9e53-c3fcdfd0a07b.png)


The same applies for all items.

Reply:
done